vite.config.ts

```diff
@@ -3,6 +3,48 @@ import vue from '@vitejs/plugin-vue'
 import vuetify, {transformAssetUrls} from 'vite-plugin-vuetify'
 import {defineConfig} from 'vite'
 import {fileURLToPath, URL} from 'node:url'
+import {type Plugin} from 'vite'
+import postcss, {Root} from 'postcss'
+
+
```

Comment:
This plugin will transform Vuetify's `@media` queries to be rem-based instead of px-based. 

However, media queries ignore the font-size set on the `<html>` (or any) element, so this won't have any effect if you resize text that way. It will work if you resize text by changing your browser's default font size.